Formula/imessage-exporter.rb

```diff
@@ -0,0 +1,22 @@
+class ImessageExporter < Formula
+  desc "iMessage conversation database export and inspector"
+  homepage "https://github.com/ReagentX/imessage-exporter"
+  url "https://github.com/ReagentX/imessage-exporter/archive/refs/tags/1.1.0.tar.gz"
+  sha256 "b932862b5f4c1c607c4dd401f99d088b4e1291ce7d5c2a7d1c40da9c91478e97"
+  license "GPL-3"
+
+  depends_on "rust" => :build
+
+  def install
+    # manifest set to 0.0.0 for some reason, matching upstream build behavior
+    # https://github.com/ReagentX/imessage-exporter/blob/develop/build.sh
+    inreplace "imessage-exporter/Cargo.toml", "version = \"0.0.0\"",
+                                              "version = \"#{version.to_s}\""
```

Comment:
Can you ask upstream to properly name their releases?

Reply:
"Proper" would be to commit the most recent released version string, and bump with a tagged commit? (Right now, upstream's own build script injects the latest tagged release when it builds downloadable releases.)